.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  PHPUnit:
```

Comment:
```suggestion
  PHPUnit:
    name: PHPUnit (PHP ${{ matrix.php }})
```

Reply:
My motivation was to keep this as minimal as possible, but I agree `PHPUnit (PHP 7.4)` could help make this more obvious than `PHPUnit (7.4)`. Updated as suggested :+1: